recipes/getopt-win32/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "getopt-win32" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libimobiledevice-win32/getopt/archive/{{ version }}.tar.gz
+  sha256: 7e9653ecd58ce4149959bf6a905f4ab2f7889856fe1218afbf84284074f9e549 
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
Add `run_exports` here?

Reply:
@isuruf I've not idea about the compatibility here. Should I put `x.x.x` ?

Reply:
Yes, that's a good idea.